FirebaseDatabase/Tests/Resources/GoogleService-Info.plist

```diff
@@ -2,27 +2,35 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
-	<key>API_KEY</key>
-	<string>correct_api_key</string>
-	<key>TRACKING_ID</key>
-	<string>correct_tracking_id</string>
 	<key>CLIENT_ID</key>
-	<string>correct_client_id</string>
+	<string>773687697675-q9dgvvkm3jfu2qjn4fgcvfvhcd7p1m61.apps.googleusercontent.com</string>
 	<key>REVERSED_CLIENT_ID</key>
-	<string>correct_reversed_client_id</string>
-	<key>GOOGLE_APP_ID</key>
-	<string>1:123:ios:123abc</string>
+	<string>com.googleusercontent.apps.773687697675-q9dgvvkm3jfu2qjn4fgcvfvhcd7p1m61</string>
+	<key>API_KEY</key>
+	<string>AIzaSyBcw9bHatsZa6tU-iHWDIFFZnS_E1k4i7A</string>
```

Comment:
Please revert these changes and add this file to .gitignore.

Reply:
Hmm, https://github.com/firebase/firebase-ios-sdk/blob/master/.gitignore#L9